src/providers/postgres/qgspostgresprovider.cpp

```diff
@@ -403,6 +403,17 @@ static bool operator<( const QVariant &a, const QVariant &b )
 }
 #endif
 
+#if QT_VERSION >= 0x050000 && QT_VERSION < 0x050600
```

Comment:
Maybe this should be defined in qgis.h? qgsVariantLessThan could benefit from using it too.